homeassistant/util/yaml.py

```diff
@@ -185,12 +185,9 @@ def _ordered_dict(loader: SafeLineLoader,
 
         if key in seen:
             fname = getattr(loader.stream, 'name', '')
-            first_mark = yaml.Mark(fname, 0, seen[key], -1, None, None)
-            second_mark = yaml.Mark(fname, 0, line, -1, None, None)
-            raise yaml.MarkedYAMLError(
-                context="duplicate key: \"{}\"".format(key),
-                context_mark=first_mark, problem_mark=second_mark,
-            )
+            _LOGGER.warning(
```

Comment:
Make this an error? Maybe even persistent notification?
Duplicate keys means the config is broken and should be fixed. Yes not starting is too extreme, but warning is easily missed.